documentation/Migration/Update-0.109.md

```diff
@@ -36,11 +36,41 @@ protected override bool IsFileSupportedCore(string path)
 
 The following are required if you are using the `Engine`:
 
+### Creation
+
+Plugins and DataSources are now loaded independently of the `Engine`. These changes were motivated by a desire to make feedback to the user about issues clearer, reduce coupling, and enable reuse. This makes the `Engine` much more powerful and flexible.
+
+- Users will now need to create a `DataSourceSet` to pass to the `Engine` via the `EngineCreateInfo` class rather than calling the `Engine.Add*` methods.
+    - See the [`Engine`](Using-the-SDK/Using-the-engine.md) documentation for more details on using `DataSourceSet`, `PluginSet`, and the `Engine` together.
+
+As a simple example:
+````cs
+using var engine = Engine.Create();
+engine.AddDataSource(new FileDataSource("test.txt"));
+engine.EnableCooker(cookerPath);
+
+// ... use the engine as desired.
+
+````
+becomes
+````cs
+
+using var dataSources = DataSourceSet.Create();
```

Comment:
the usings in this and above don't have corresponding closure scopes. should they?

Reply:
For newer versions of C#, they don't need them. However, for consistency with other docs, I will put them in. Thanks!